### PR TITLE
Reapply "Jupiter 5.7.0" 

### PR DESCRIPTION
**Pull request by hmusum**
Reverts vespa-engine/vespa#15054

### CODE REVIEW

**Review by jonmv — APPROVED** [review]

**Comment by hmusum** [comment]
@bjorncs could you comment on the change in  tenant-cd-api/pom.xml?  Is it OK to bump the version here and in internal repo or does the comment indicate that more needs to be done?

File: tenant-cd-api/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
             This version must match the string in all ExportPackage annotations.
```
Comment (bjorncs) [inline tenant-cd-api/pom.xml:25]: @jonmv Read this!

**Review by bjorncs — CHANGES_REQUESTED** [review]
See comment

File: tenant-cd-api/pom.xml (hunk 1 above)
Comment (jonmv) [reply tenant-cd-api/pom.xml:25]: Thanks!

Comment (hmusum) [reply tenant-cd-api/pom.xml:25]: Where are the ExportPackage annotations we are talking about here?

Comment (bjorncs) [reply tenant-cd-api/pom.xml:25]: In the package-info.java files in tenant-cd-api (for the the junit annotations).

Comment (hmusum) [reply tenant-cd-api/pom.xml]: Updated (although the old version exported was 5.4.2, not the expected 5.6.2)

**Review by bjorncs — APPROVED** [review]

**Comment by bjorncs** [comment]
FYI @tokle

**Pull request merged**